homeassistant/components/maxcube.py

```diff
@@ -28,10 +28,12 @@
 NOTIFICATION_TITLE = 'Max!Cube gateway setup'
 
 CONF_GATEWAYS = 'gateways'
+CONF_UPDATE_INTERVAL = 'update_interval'
```

Comment:
I'm not sure about this configuration key name. Most platforms use [`scan_interval`](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval) for an update/scan interval.

IMO `update_interval` makes more sense, but `scan_interval` would be much more consistent.

Reply:
I have changed it to scan_interval to be more consistent